guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/java/org/guvnor/m2repo/client/widgets/ArtifactListViewImpl.java

```diff
@@ -46,11 +43,6 @@
 
     protected ArtifactListPresenter presenter;
 
-    @Inject
-    protected Caller<M2RepoService> m2RepoService;
-
-    protected String currentFilter = "";
```

Comment:
View doesn't need to know about the filter. This responsibility has been moved to data provider.